util/stmtsummary/evicted.go

```diff
@@ -51,6 +56,14 @@ func newStmtSummaryByDigestEvictedElement(beginTime int64, endTime int64) *stmtS
 		beginTime:    beginTime,
 		endTime:      endTime,
 		digestKeyMap: make(map[string]struct{}),
+		otherSummary: &stmtSummaryByDigestElement{
+			beginTime:    beginTime,
+			endTime:      endTime,
+			authUsers:    make(map[string]struct{}),
+			minLatency:   time.Duration(math.MaxInt64),
+			backoffTypes: make(map[string]int),
+			firstSeen:    time.Now(),
```

Comment:
Should `firstSeen` be initialized now? When `otherSummary` is created, maybe there's no SQL to be added. When a SQL comes in, its time must be greater than this `firstSeen`.

Reply:
done, @djshow832 PTAL.